app/services/foreman_ansible/ansible_info.rb

```diff
@@ -1,16 +1,18 @@
 module ForemanAnsible
   class AnsibleInfo < ::HostInfo::Provider
-    def host_info
-      { 'parameters' => ansible_params }
+    def host_info(redact_secrets = false)
+      { 'parameters' => ansible_params(redact_secrets) }
     end
 
-    def ansible_params
+    def ansible_params(redact_secrets = false)
```

Comment:
If the value of `redact_secrets` is based on the user's permissions, why do we need to pass `redact_secrets` to this method? Can't we check if the user has the necessary permissions directly within the method instead?